weasyl/controllers/user.py

```diff
@@ -109,9 +111,9 @@ def signup_post_(request):
     form = request.web_input(
         username="", password="", passcheck="", email="", emailcheck="",
         day="", month="", year="", recaptcha_challenge_field="",
-        g_recaptcha_response=request.web_input("g-recaptcha-response"))
+        g_recaptcha_response="")
```

Comment:
This breaks the login form, as an FYI. https://paste.weasyldev.com/show/nPaojKrYSILAx5JawNwS/

I don't have time enough at the moment for a deep-dive fix on this, but the ``request.web_input(...)`` was in there because of compile issues, since the reCAPTCHA code uses dashes, and dashes don't/didn't play well with the code when I initially implemented the v2 code; simply put, there's no field on the form that is named ``g_recaptcha_response``. It will be named ``g-recaptcha-response`` when the JavaScript code loads.